tableaudocumentapi/connection.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, connxml):
         self._dbname = connxml.get('dbname')
         self._server = connxml.get('server')
         self._username = connxml.get('username')
+        self._authentication = connxml.get('authentication')
+        self._class = connxml.get('class')
+
+    def __repr__(self):
```

Comment:
why represent this in a "made-up" style instead of simply representing it as it would look in xml?